app/src/main/res/layout/main_fragment.xml

```diff
@@ -132,21 +141,21 @@
         <LinearLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+            android:background="#FFFFFFFF"
             android:orientation="vertical">
 
             <TextView
-                android:layout_width="wrap_content"
+                android:layout_width="402dp"
                 android:layout_height="wrap_content"
-                android:textColor="#000"
-                android:textSize="16dp"
-                android:layout_margin="10dp"
-                android:text="Country Cases" />
+                android:background="#5B000000"
+                android:text="Country Cases"
+                android:textColor="#0B0000"
+                android:textSize="16dp" />
```

Comment:
@holtcl we are currently keeping the UI of an application to a minimal level. We will start making it look aesthetically clean in the future. Don't need this right now.